config/brakeman.ignore

```diff
@@ -1,26 +1,126 @@
 {
   "ignored_warnings": [
     {
-      "warning_type": "Command Injection",
-      "warning_code": 14,
-      "fingerprint": "6bb72563651a196605533cf18c0ba1bcb6acc357b2a6c60650491d6de7a17c11",
-      "check_name": "Execute",
-      "message": "Possible command injection",
-      "file": "lib/guard/lint.rb",
-      "line": 13,
-      "link": "https://brakemanscanner.org/docs/warning_types/command_injection/",
-      "code": "system(\"#{\"bundle exec govuk-lint-ruby\"} #{files.join(\" \")}\")",
+      "warning_type": "SQL Injection",
+      "warning_code": 0,
+      "fingerprint": "412ff9036816bcf66b884fd927c1416c07c6cc25a7623eb2ea8617e6259d757e",
+      "check_name": "SQL",
+      "message": "Possible SQL injection",
+      "file": "app/domain/finders/content.rb",
+      "line": 67,
+      "link": "https://brakemanscanner.org/docs/warning_types/sql_injection/",
+      "code": "scope.where(\"#{view[:table_name]}.dimensions_month_id = '#{sprintf(\"%s-%02d\", year, Date::MONTHNAMES.index(month.capitalize))}'\").joins(\"JOIN dimensions_editions ON #{view[:table_name]}.dimensions_edition_id = dimensions_editions.id\")",
       "render_path": null,
       "location": {
         "type": "method",
-        "class": "Guard::Lint",
-        "method": "run_on_modifications"
+        "class": "Finders::Content",
+        "method": "find_by_month"
       },
-      "user_input": "files.join(\" \")",
+      "user_input": "view[:table_name]",
+      "confidence": "Weak",
+      "note": "month_id is an integer, view[...] is a hard-coded string; not user input."
+    },
+    {
+      "warning_type": "SQL Injection",
+      "warning_code": 0,
+      "fingerprint": "45c97e3fbe33b7415c7e1b81d39f828f114131304aa46d3bc96efb8157ee3ccb",
+      "check_name": "SQL",
+      "message": "Possible SQL injection",
+      "file": "app/domain/healthchecks/etl_metric_values.rb",
+      "line": 31,
+      "link": "https://brakemanscanner.org/docs/warning_types/sql_injection/",
+      "code": "Facts::Metric.for_yesterday.where(\"#{metric} > 0\")",
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "Healthchecks::EtlMetricValues",
+        "method": "number_of_metric_values"
+      },
+      "user_input": "metric",
+      "confidence": "Medium",
+      "note": "this is called with hard-coded strings, not user input"
```

Comment:
I see so this is the fingerprint file of "ones Brakeman would have flagged on a fresh scan, but where we've put a `note` sayng we think this is OK?